src/utils/constants.js

```diff
@@ -1,6 +1,14 @@
 const carouselList = document.querySelectorAll('.carousel');
 const donateBtn = document.querySelector('.donations__submit-button');
 const formElement = document.querySelector('.donations__form');
+const nameInput = formElement.elements.name
+const surnameInput = formElement.elements.surname
+const emailInput = formElement.elements.email
+const donationAmount = formElement.elements["donation-amount"]
```

Comment:
Since we don't want a mixture of double quotes and single quotes, can you change all your double quotes to single?